packages/plugin-ua-parser-browser/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@amplitude/plugin-ua-parser-browser",
```

```suggestion
  "name": "@amplitude/plugin-user-agent-enrichment-browser",
```

i suggest that we change the name to something referring to an user agent enrichment than user agent parser. the parsing is internal implementation and and the value this plugin provide is the enrichment portion and not the parser portion. i'd also avoid using abbreviations like ua since it makes our plugin less discoverable